csfieldguide/templates/interactives/pixel-viewer.html

```diff
@@ -39,6 +39,21 @@ <h5>{% trans "Options" %}</h5>
                     <input id="pixel-viewer-interactive-show-pixel-fill" type="checkbox" checked="checked">
                     {% trans "Show pixel background" %}
                 </label>
+                <div class="mb-1">
+                    {% trans "Colour code: " %}
+                    <div class="form-check form-check-inline">
+                        <input class="form-check-input" type="radio" name="colourCode" id="rgb-colour-code" value="rgb" checked>
+                        <label class="form-check-label" for="rgb-colour-code">{% trans "Decimal (as separate RGB values)" %}</label>
```

Comment:
maybe just `(separate RGB)` for the two for concision. I think they can be stacked to look a bit nicer
![image](https://user-images.githubusercontent.com/25916475/86544272-93369f80-bf79-11ea-946a-817e6d8fd96a.png)